src/bci_build/package/appcontainers.py

```diff
@@ -169,7 +169,13 @@
                     regex_in_build_description="%%mariadb_version%%",
                     package_name="mariadb",
                     parse_version="minor",
-                )
+                ),
+                Replacement(
+                    file_name="README.md",
+                    regex_in_build_description="%%mariadb_version%%",
+                    package_name="mariadb",
+                    parse_version="minor",
+                ),
```

Comment:
actually this part seems unnecessary now, there is no `%%mariadb_version%%` as far as I can see

Reply:
true, I missed it

Reply:
It is still used, the URL created is `registry.suse.com/suse/rmt-mariadb:%%mariadb_version%%`.

The property `pretty_reference` uses the `version` attribute, which is set to `%%mariadb_version%%`.

Reply:
Ah thank you.